jx_sqlite/query_table.py

```diff
@@ -124,7 +134,7 @@ def query(self, query):
 
         result = self.db.query(command)
 
-        column_names = query.edges.name + query.groupby.name + listwrap(query.select).name
+        column_names = listwrap(query.edges).name + listwrap(query.groupby).name + listwrap(query.select).name
```

Comment:
Previously when at the end of elif block when ` query.edges = None` , a test raised an 
error : None type has no attribute name. But now with this swap , it's working good.